Makefile

```diff
@@ -27,6 +27,8 @@ ci: build test
 test: ## Test the project
 	GO111MODULE=on go test ./...
 
+ci-lint: lint fmt vet
+
```

Comment:
Any reason not to add this to the ci target? 

```
ci: lint fmt vet build test
```

Devs can just run them individually locally, most often `make lint` is run.